### PR TITLE
Fix model version parsing after #263

### PR DESCRIPTION
**Pull request by rmunn**
Now that model versions are doubles, we need to change some of the int.Parse calls in LfMergeBridge.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/283)

### CODE REVIEW

File: src/LfMergeBridge/LanguageForgeMakeCloneActionHandler.cs [hunk 1]
```diff
@@ -109,7 +109,7 @@ private static void FinishClone(IProgress progress, ref string somethingForClien
 			}
 			// See if repo has higher branch than LF called for.
 			var highestHead = LfMergeBridgeUtilities.GetHighestRevision(hgRepository);
-			if (int.Parse(highestHead.Branch) > int.Parse(desiredBranchName))
+			if (double.Parse(highestHead.Branch) > double.Parse(desiredBranchName))
```
Comment (ermshiperete) [inline src/LfMergeBridge/LanguageForgeMakeCloneActionHandler.cs:112]: Probably use `GetModelVersionFromBranchName` here as well?

**Comment by jasonleenaylor** [comment]
I'm confused about this PR. Why is lfmerge so far behind develop? Aren't all the necessary changes already on the develop branch?

**Comment by rmunn** [comment]
Back when I last looked at the FLExBridge code, we were building the LfMergeBridge code on a separate `lfmerge` branch. I saw that it had been deleted from the FLExBridge GitHub repo, so I restored it before adding this patch, because I assumed that the `master` and `develop` branches did not contain the LfMergeBridge code. But if it was merged into `develop` at some point during the year I was on furlough and not paying attention to the repo, then I'll do a build of LfMergeBridge from the `develop` branch and see if that solves the problem.

**Comment by rmunn** [comment]
Closing; will open a new PR against `develop` branch with the actual changes that need to happen.

**Comment by rmunn** [comment]
Replaced and superseded by #285.

**Pull request closed without merging**